app/src/main/res/values/styles.xml

```diff
@@ -464,4 +464,13 @@
   <style name="TextViewCenterHorizontal" parent="TextView.Common1">
     <item name="android:gravity">center_horizontal</item>
   </style>
+
+  <!-- ShapeableImageView -->
+  <!-- The corner size & family type are set up to ensure a circular shape is created for components
+   with equal width & height.
+   (per https://material.io/design/shape/about-shape.html#shape-customization-tool). -->
+  <style name="ShapeAppearanceOverlay.App" >
```

Comment:
Given that we're determining this name, perhaps we can make it more descriptive. What about: ``ShapeableImageView.RoundedShape`` (or something similar)?

Reply:
> Given that we're determining this name, perhaps we can make it more descriptive. What about: `ShapeableImageView.RoundedShape` (or something similar)?

It seems like we need to stick to the 'ShapeAppearanceOverlay' name here but I added the 'RoundedShape' as you suggested.